compiler/src/yul/runtime/functions/abi.rs

```diff
@@ -10,6 +10,11 @@ use fe_analyzer::namespace::types::{
 };
 use yultsur::*;
 
+// Return all abi runtime functions
```

Comment:
Can you change the comments to triple slashes `///` to make sure they are picked up as API docs by rustdoc (Applies to all the added comments in this PR.